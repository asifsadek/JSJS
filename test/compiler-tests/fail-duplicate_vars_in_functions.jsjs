val temp = /\(x) => {
  val x = "inner";
  x;
};

print(temp("yo"));
