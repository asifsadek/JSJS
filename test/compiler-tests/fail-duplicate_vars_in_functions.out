Error: 'x' cannot be redefined in the current scope
